### PR TITLE
Update JvDBGridExport.pas

### PR DESCRIPTION
**Pull request by RegentGaudreau**
Added a HeaderRow property to easily specify if we want the row 0 or not

### CODE REVIEW

**Comment by obones** [comment]
What's the intent with this pull request?

**Comment by RegentGaudreau** [comment]
Hi,

I am using JVCL, and JvDBGridExport

Sometime i do not want the header in the export and there was no way of 
doing this, thats why i have added the HeaderRow property for this 
option.

other than that, i just wanted to submit my change to you, and add it to 
the master source; sorry if this is not the way to do ?

Many thanks

Regent

------ Message d'origine ------
De: "obones" <notifications@github.com>
À: "project-jedi/jvcl" <jvcl@noreply.github.com>
Cc : "Redge" <regent@air-g.net>; "Author" <author@noreply.github.com>
Envoyé : 22/09/2020 09:46:20
Objet : Re: [project-jedi/jvcl] Update JvDBGridExport.pas (#147)

**Comment by obones** [comment]
It is the proper way, just that the commit message was not clear at all.

**Comment by RegentGaudreau** [comment]
Ok, thanks; i will try to put more details next time.

------ Message d'origine ------
De: "obones" <notifications@github.com>
À: "project-jedi/jvcl" <jvcl@noreply.github.com>
Cc : "Redge" <regent@air-g.net>; "Author" <author@noreply.github.com>
Envoyé : 22/09/2020 09:56:26
Objet : Re: [project-jedi/jvcl] Update JvDBGridExport.pas (#147)

**Comment by RegentGaudreau** [comment]
before Adding; i just remembered to set the defautl value for FRowHeader 
to True;

line 592  FHeaderRow := True;


------ Message d'origine ------
De: "obones" <notifications@github.com>
À: "project-jedi/jvcl" <jvcl@noreply.github.com>
Cc : "Redge" <regent@air-g.net>; "Author" <author@noreply.github.com>
Envoyé : 22/09/2020 09:56:26
Objet : Re: [project-jedi/jvcl] Update JvDBGridExport.pas (#147)

File: jvcl/run/JvDBGridExport.pas [hunk 1]
```diff
@@ -178,6 +179,7 @@   TJvDBGridExcelExport = class(TJvCustomDBGridOleExport)
     property Visible: Boolean read FVisible write FVisible default False;
     property Orientation: TWordOrientation read FOrientation write FOrientation default woPortrait;
     property AutoFit: Boolean read FAutoFit write FAutoFit;
+    property HeaderRow: Boolean read FHeaderRow write FHeaderRow;
```
Comment (ronaldhoek) [inline jvcl/run/JvDBGridExport.pas:182]: Specify the default here too:

`property HeaderRow: Boolean read FHeaderRow write FHeaderRow default True;`

Comment (RegentGaudreau) [reply jvcl/run/JvDBGridExport.pas]: done, thanks

**Pull request merged**